### PR TITLE
Add vue component for sidebar alpha tab

### LINKED ISSUE #1450: Design the Alphabetical list component for sidebar
## Description of the enhancement
The alphabetical listing should be implemented as a Vue component

## What new functionalities would the enhancement make possible?
This is the first prototype of the partial page loading for the concept page
This also is one of the first cases for building front-end tests

### PR DESCRIPTION
**Pull request by UnniKohonen**
## Reasons for creating this PR

Adding the alphabetical index tab to sidebar

## Link to relevant issue(s), if any

- Closes #1450 

## Description of the changes in this PR

- Add a Vue component for the alphabetical index
- Move other Vue script files to scripts.inc
- Add bootstrap script to scripts.inc
- Partial page load for concept page

## Checklist

- [ ] phpUnit tests pass locally with my changes
- [ ] I have added tests that show that the new code works, or tests are not relevant for this PR (e.g. only HTML/CSS changes)
- [ ] The PR doesn't reduce accessibility of the front-end code (e.g. tab focus, scaling to different resolutions, use of `.sr-only` class, color contrast)
- [ ] The PR doesn't introduce unintended code changes (e.g. empty lines or useless reindentation)

### CODE REVIEW

File: src/view/scripts.inc [hunk 1]
```diff
@@ -40,5 +40,14 @@ const SKOSMOS = {
...
+<script src="node_modules/bootstrap/dist/js/bootstrap.bundle.js"></script>
+
 <!-- Vue.js -->
 <script src="node_modules/vue/dist/vue.global.js"></script>
+
+<!-- Vue.js components -->
+<script src="resource/js/vocab-counts.js"></script>
+<script src="resource/js/term-counts.js"></script>
+<script src="resource/js/concept-mappings.js"></script>
+<script src="resource/js/tab-alpha.js"><script>
```
Comment (osma) [inline src/view/scripts.inc:53]: Newline missing

File: src/view/sidebar.inc [hunk 2]
```diff
@@ -48,6 +56,23 @@
...
+      <div role="tabpanel" class="tab-pane" id="tab-groups">
+        <p>ccc</p>
+      </div>
+      <div role="tabpanel" class="tab-pane" id="tab-changes">
+        <p>ddd</p>
+      </div>
+    </div>
+
+
     {% if letters %}
```
Comment (osma) [inline src/view/sidebar.inc:76]: Remove this `if letters` block (lines 51-64) as the functionality was reimplemented in Vue

**Review by osma — CHANGES_REQUESTED** [review]
I commented on a few minor changes that should be done before merging.

The details of partial page loading still need some more work. They could be done in this PR if it's easy, but an alternative is to postpone this and create a new issue for partial page loads.

File: resource/js/tab-alpha.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+          for (let i = 0; i < toBeRemoved.length; i++) {
+            mainContent.removeChild(toBeRemoved[i])
+          }
+
+          // inserting concept card into vocab info
+          for (let i = 0; i < conceptMainContent.length; i++) {
+            mainContent.prepend(conceptMainContent[i])
+          }
+
+          // what other properties should be changed?
```
[the remark this answers is not in the document]
Comment (UnniKohonen) [reply resource/js/tab-alpha.js:121]: I will make a separate issue for this

**Pull request merged**